kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerHttpException.java

```diff
@@ -16,6 +16,7 @@
 
 package com.netflix.spinnaker.kork.retrofit.exceptions;
 
+import com.netflix.spectator.impl.Preconditions;
```

Comment:
Let's use
```suggestion
import com.google.common.base.Preconditions;
```
instead.

Reply:
com.google.common.base.Preconditions instead of com.netflix.spectator.impl.Preconditions for this added dependency com.google.guava:guava in kork-retrofit module